pom.xml

```diff
@@ -141,6 +141,7 @@
   <modules>
     <module>cdap-common-unit-test</module>
     <module>cdap-api</module>
+    <module>cdap-app-templates</module>
```

Comment:
Why is this added? cdap-app-templates build is triggered only when "templates" profile is enabled. Similar to examples.